app/jobs/scheduled/update_user_counts_for_chat_channels.rb

```diff
@@ -15,8 +15,8 @@ def set_user_count(chat_channel)
       new_count = chat_channel
         .user_chat_channel_memberships
         .joins(:user)
+        .where(user: { active: true, suspended_till: nil, staged: false })
```

Comment:
Hmm I think we should still use the `merge`s here to reuse scopes from core, we almost have all the ones we need:

https://github.com/discourse/discourse/blob/a258630790d7c976632d857aa9eaf6e09fd71d4e/app/models/user.rb#L221-L222

Would just need to add a `:not_staged` scope to core and use that as well. Alternatively, add a new scope that is `active_not_suspended` or something that does all 3,

Reply:
Okay, sounds good. I was wondering if we had an existing scope for this, I guess I'll create a new one then.